src/main/resources/com/puppycrawl/tools/checkstyle/checks/coding/messages_fr.properties

```diff
@@ -35,7 +35,7 @@ missing.ctor=Il manque un constructeur à la classe.
 missing.package.declaration=Déclaration de package manquante.
 missing.super.call=La méthode ''{0}'' devrait appeler ''super.{0}''.
 missing.switch.default=Il manque le cas "default" dans le bloc "switch".
-missing.switch.nullcase=Switch utilisant des types de r�f�rence doit avoir un cas nul.
+missing.switch.nullcase=Switch utilisant des types de r f rence doit avoir un cas nul.
```

Comment:
Same.

Reply:
done

Reply:
please see there is still something weird.

```
-missing.switch.nullcase=Switch utilisant des types de référence doit avoir un cas nul.
+missing.switch.nullcase=Switch utilisant des types de référence doit avoir un cas nul.
```

what is the difference

Reply:
I am not sure but git diff doesn't lie. it may be something with the whitespace you can only see with a hex editor.

For example, Japanese character set has a special whitespace which doesn't match the ASCII whitespace.

Reply:
@nrmancuso Please help, how did you fix it before? In https://github.com/checkstyle/checkstyle/pull/15358#discussion_r1694261409

copy/paste from old branch is not working it substitutes the non-ascii with space or git diff produces differences even if the text is identical. very weird and frustrating

Reply:
In the FR file, `référence` is the issue.

Your PR has the hex:
`72 C3 A9 66 C3 A9 72 65 6E 63 65`
Master has the hex:
`72 E9 66 E9 72 65 6E 63 65`

According to ChatGPT ( and I agree with it's findings) :
> The sequence 72 C3 A9 66 C3 A9 72 65 6E 63 65 is the UTF-8 encoding, where each 'é' is represented by the two-byte sequence C3 A9.
> The sequence 72 E9 66 E9 72 65 6E 63 65 is the Latin-1 encoding, where each 'é' is represented by the single byte E9.
> In both cases, the byte sequences represent the same string "référence", but they are encoded differently according to the specified character encoding schemes.

Whatever you are using to edit these files is changing the encoding.

Reply:
I noticed this kind of updates on one of previous PRs, Nick unintentionally damaged Unicode, root reason was not found, we just reversed and moved forward.

Reply:
done on GH